src/mongocxx/bulk_write.hpp

```diff
@@ -101,6 +101,14 @@ class MONGOCXX_API bulk_write {
     MONGOCXX_DEPRECATED void* implementation() const;
 
     ///
+    /// Whether or not to bypass document validation for this operation.
+    ///
+    /// @param bypass_document_validation
+    ///   Whether or not to bypass document validation.
+    ///
+    void bypass_document_validation(bool bypass_document_validation);
```

Comment:
Normally, I'm pretty against bool parameters to functions, but I'm ok with it on a trivial setter when there are no other parameters. No need to change anything, just pointing out that I view this as a special exception.